vividus-plugin-mail/src/main/resources/properties/defaults/defaults.properties

```diff
@@ -0,0 +1,4 @@
+mail.service.polling.duration=PT30S
+mail.service.polling.retry-times=5
+mail.service.polling.wait-message-event-duration=PT5S
+mail.service.folder=Inbox
```

Comment:
word `service` doesn't bring much sense, I think it should be replaced with `imap`